Presentation.Web/app/components/it-system/it-system-overview.controller.ts

```diff
@@ -493,7 +493,27 @@
                         .withoutSorting()   //NOTICE: NO sorting OR filtering on computed field!
                         .withInitialVisibility(false)
                         .withRendering(dataItem => Helpers.RenderFieldsHelper.renderDate(dataItem.ActiveArchivePeriodEndDate))
-                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderDate(dataItem.ActiveArchivePeriodEndDate)));
+                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderDate(dataItem.ActiveArchivePeriodEndDate)))
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("RiskSupervisionDocumentationName")
+                        .withTitle("Risikovurdering")
+                        .withId("riskSupervisionDocumentationUrlName")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
+                        .withInitialVisibility(false)
+                        .withContentOverflow()
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderReference(dataItem.RiskSupervisionDocumentationName, dataItem.RiskSupervisionDocumentationUrl))
+                        .withExcelOutput(dataItem => Helpers.ExcelExportHelper.renderUrlOrFallback(dataItem.RiskSupervisionDocumentationUrl, dataItem.RiskSupervisionDocumentationName)))
+                .withColumn(builder =>
+                    builder
+                        .withDataSourceName("LinkToDirectoryName")
+                        .withTitle("Fortegnelse")
+                        .withId("LinkToDirectoryUrlName")
+                        .withFilteringOperation(Utility.KendoGrid.KendoGridColumnFiltering.Contains)
+                        .withInitialVisibility(false)
+                        .withContentOverflow()
+                        .withRendering(dataItem => Helpers.RenderFieldsHelper.renderReference(dataItem.LinkToDirectoryName, dataItem.LinkToDirectoryUrl))
```

Comment:
Eftersom det ikke kun er referencer der bliver renderet af denne skal navnet måske rettes til? Noget ala renderNameOrUrl?

Reply:
Ja det kan man sige men i kitos er en ekstern reference altid et link og optionelt et navn.. det dækker begrebet over. Det er vist også derfor der er render metoder med forskelligt abstraktionsniveau. Jeg ser lige om den kan renames iden at lave for meget ulykke